.gitlab-ci.yml

```diff
@@ -826,21 +825,23 @@ workflow:
     allow_failure: true
 
 .on_system_probe_changes_or_manual:
```

Comment:
Can we retain the ability to run manually?

Reply:
Since it will always run now, it will simply not execute if there's not change. So technically we can choose to re-run the job but it will not get executed because there's no change registered. So as it is we lost the abillity to run manually.
A current way if we want to keep the ability to run manually would be to have duplicate of the jobs that extend the same base but with a rule set as manual instead of the rule. What do you think about that way ?